### DISCUSSION TITLE
10%fitting error

### DISCUSSION THREAD

**Discussion by lclclc1225** [ttadano/alamode#229] [Q&A]
Hello, when I calculate the second order force constant, the fitting error is about 10%, the methods on the official website are aimed at more than 90% of the error, may I ask how to reduce the fitting error of about 10%.

**Comment by ttadano** [comment]
Hi,

Did you use the `--offset` option described in the F&Q section of the document? If not, please try it.

If the fitting error is still as large as 10% with the `--offset` option, I would suggest that you decrease the displacement magnitude to a smaller value.
